### DISCUSSION TITLE
Lost access to my account after loosing my recovery devices

### DISCUSSION THREAD

**Discussion by SiroDevs** [community/community#55254] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hello @Github I need help with recovery of my account

https://github.com/jacksiroke

My phone which had the Authentication app was stolen late last year together with my laptop which had the recovery codes. I need help to recover my Github acc as I am locked out if completely. I gave up getting my laptop back last month. 

I would be happy to find a way around this issue. I have been using my other account but now I need access account to my main account.

**Comment by Nayetwolf** [comment]
That does sound like a tough spot, and I'm sorry to hear about the situation. Losing your recovery devices is frustrating as heck, especially since you count on your GitHub account for some important projects.

The best course of action is to reach out to GitHub Support directly. You can submit a support request through their help page, detailing your situation. You can say something like, "My GitHub username is X; I lost access to my authentication app and recovery codes and have been using X account in the meantime until I get this sorted.".

Also, please include in your request as much ancillary information as possible that would help in verifying your identity; this could include past email exchanges with the GitHub team or specific details regarding how you use your account. Keep in mind that GitHub security turns most requests around so that identification can be positively confirmed before reinstatement is allowed.

Just keep in mind, recovery can rightfully sometimes take some time, so be patient and let them work through your request. Good luck; hopefully you can gain access to your account again soon!
